src/main.js

```diff
@@ -55,21 +55,20 @@ let loud = false;
 if (typeof window !== "undefined") {
   runtimeConfig = paramsToObj(window.location.search);
   loud = runSafe = !!window.location.search; // anything passed!
+  runSafe && personinfo.tryLocalStorage();  // 1st attempt, 2nd during mainloop.  see #212
 }
 
-
-// will only catch things from other windows
-//if (window) {
-//  window.addEventListener("storage", function (e) {
-//    log("storage", key, oldValue, newValue, uri);
-//  }, false);
-//}
-
-
 // is there a timer here? I dunno!
 let mainloop = function (repairsList) {
   loud && actions.log("heartbeat main loop");
-  runner.runAll(repairsList,
+  personinfo.tryLocalStorage();  // see #212.
+  let err = "first time run, cannot guarantee persistent localStorage (see #212)";
+  if (!personinfo.isLocalStoragePersistent()) {
```

Comment:
Maybe I misunderstand.  Recipes in this system will NEVER run on first view / run of the code.  All recipes need to remember some state (that they ran).  If this changes, I can move it deeper into each recipe.